test/e2e/specs/editor/various/embedding.spec.js

```diff
@@ -194,7 +194,8 @@ test.describe( 'Embedding content', () => {
 		embedUtils,
 	} ) => {
 		await embedUtils.interceptRequests( {
-			'https://twitter.com/notnownikki/': MOCK_CANT_EMBED_RESPONSE,
+			'https://twitter.com/notnownikki/':
+				MOCK_BAD_EMBED_PROVIDER_RESPONSE,
```

Comment:
The `MOCK_CANT_EMBED_RESPONSE` returns a fallback `preview.html`, causing the side-effect to be skipped.